migrate/0004_modify_rotations.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'sequel'
+require_relative '../models/action'
+
+rotations = {
+  '7' => 5,
+  '8' => 4,
+  '12' => 5,
+  '16' => 4,
+  '19' => 3,
+  '206' => 3,
+  '45' => 3,
+  '46' => 3,
+  '47' => 2,
+  '465' => 2,
+}.freeze
+
+Sequel.extension(:pg_json_ops)
+json = Sequel.pg_jsonb_op(:action)
+
+Sequel.migration do
+  up do
+    rotations.each do |tile, rotation|
+      actions = Action.where(json.get_text("type") => "lay_tile",
+                             json.get_text("tile") => /^#{tile}-/).all
+
+      actions.each do |action|
+        parsed = JSON.parse(action.action)
+        current = parsed["rotation"] || 0
```

Comment:
why would there be no rotation

Reply:
I don't know, got this when downgrading the migration locally, don't even know if you want a migration?